hedera-node/docs/design/services/smart-contract-service/update_token_custom_fees_sc.md

```diff
@@ -0,0 +1,87 @@
+# Update Token Custom Fees via Smart Contracts
+
+## Purpose
+
+In order to support updating the Token Custom Fees via smart contract calls, we need to expose new smart contract functions in the Hedera Smart Contract Service (HSCS) logic exposing Hedera Token Service (HTS) logic. This document will define the architecture and implementation of the `updateFungibleTokenCustomFees` and `updateNonFungibleTokenCustomFees` smart contract functions that will extend tha capabilities of the HSCS.
+
+## References
+
+[HIP-18](https://hips.hedera.com/hip/hip-18) - HIP that introduces token custom fees.\
+[HIP-206](https://hips.hedera.com/hip/hip-206) - HIP that enables the precompiled system contracts for Hedera Token Service.\
+[HIP-514](https://hips.hedera.com/hip/hip-514) - HIP that introduces token management via smart contracts.
+
```

Comment:
There should be a reference to HIP-1010 here!  Obviously now it is just a draft - but you know what the eventual URL will be ...

Reply:
no longer in draft.  it's been merged I believe.

Reply:
Thanks! You're totally right here! I added the link to the HIP.